### ISSUE TITLE
Auto-completion when UserId is not specified for customsessionId (for HeadlessClient)

### ISSUE THREAD

**Issue by kazu0617** [Neos-Metaverse/NeosPublic#3242]
### Is your feature request related to a problem? Please describe.

When creating Config.json, you can dynamically assign login IDs and passwords from the startup arguments without entering them in Config.json (--user --password).
However, due to the specification of customsessionId, the system will not start unless a specific UserId is specified, which is incompatible with this configuration method.

### Describe the solution you'd like

When the customsessionid is checked at startup, if it does not have a: in it, please auto-complete the UserId in front of it.

For example:
```json
{
    "loginCredential": "kazuheadless",
    "loginPassword": "xxxxxxxxxxxx",
    "startWorlds": [
      {
        "customSessionId": "Test1",
        "loadWorldPresetName": "SpaceWorld"
      }
   ]
}
```
if launch with this Config, sessionId is "S-U-kazuheadless:Test1".

### Describe alternatives you've considered

Basically, no.

We are currently working on a dedicated Headless Lacuncher, but for the time being, there is a possibility of incorporating a system that modifies Config.json to automatically set the UserId for customsessionid.

### Additional context

double checked with U-imotchi0112-vr / U-ChameleonO2 and U-kiyomaru and written by U-kazu.
